doc/extension/Pipfile

```diff
@@ -0,0 +1,12 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+taipy-gui = ">=2.0"
+
+[dev-packages]
+
+[requires]
+python_version = "3"
```

Comment:
`>= 3.8` ?

Reply:
I guess that makes a lot of sense. Hopefully this syntax is allowed...

Reply:
It is not. Only a specific version (3.8, 3.9...) is.
Therefore... 3 works here